browser/src/map/Clipboard.js

```diff
@@ -708,14 +708,15 @@ L.Clipboard = L.Class.extend({
 	},
 
 	// ClipboardContent.getType() callback
-	_navigatorClipboardGetTypeCallback: function(clipboardContent, blob, type) {
+	_navigatorClipboardGetTypeCallback: async function(clipboardContent, blob, type) {
 		var that = this;
 		if (type == 'image/png') {
 			this._pasteTypedBlob(type, blob);
 			return;
 		}
 
-		blob.text().then(function(text) {
+		try {
+			var text = await blob.text();
 			if (type === 'text/html' && text.substring(0, 4) === '<img') {
 				// Got an image, work with that directly.
 				clipboardContent.getType('image/png').then(function(blob) {
```

Comment:
you could use `await` here too

Reply:
I agree, however I also think one thing that `await` gives us is the ability to have a nicer flow and reduce nesting, so I've refactored the whole function to hopefully get more benefit from `await` rather than to use it for the sake of using it